packages/frontend/src/components/SettingsValidator/index.tsx

```diff
@@ -84,7 +87,10 @@ const SettingsValidator: FC<{ projectUuid: string }> = ({ projectUuid }) => {
                                 : null}
                         </Text>
                         <Button
-                            onClick={() => validateProject()}
+                            onClick={() => {
+                                setIsValidating(true);
+                                validateProject();
+                            }}
                             loading={isValidating}
                             disabled={isValidating}
```

Comment:
we can remove this `disabled` so it shows the spinner, my bad for introducing it!

Reply:
this can be done on another PR btw

Reply:
I think it is ok  as it is , we don't want to allow people to click on this button multiple times. Perhaps we should make disabled less `light-gray` ...

Reply:
when it's loading it's disabled from the example here: 
https://mantine.dev/core/button/#loading-state